examples/bigcommerce-catalyst/package.json

```diff
@@ -17,7 +17,7 @@
     "@bigcommerce/catalyst-client": "^0.1.1",
     "@graphql-typed-document-node/core": "^3.2.0",
     "@icons-pack/react-simple-icons": "^9.3.0",
-    "@makeswift/runtime": "^0.18.0",
+    "@makeswift/runtime": "^0.19.0",
```

Comment:
The gotcha with the `bigcommerce-catalyst` example is that it's the only example where we store `package-lock.json` in the repo, so we need to update that as well.

Reply:
Good catch, thank you. I should have been using `npm` when upgrading the runtime instead of `pnpm`